xcode/Subconscious/Shared/Components/Common/Recovery/RecoveryModeExplainPanelView.swift

```diff
@@ -11,14 +11,13 @@ import ObservableStore
 
 struct RecoveryModeExplainPanelView: View {
     var store: ViewStore<RecoveryModeModel>
-    var did: Did?
-    var onCancel: () -> Void
```

Comment:
We can send this action to the store instead of requiring the parent to pass it down.